docs/fixes/2026-07-24-describe-affected-repo-copy-race.md

```
@@ -0,0 +1,72 @@
+# Fix: flaky repo-copy race in describe-affected tests
+
+**Date:** 2026-07-24
+
+## Summary
+
+The Linux Acceptance Tests job failed with `TestDescribeAffectedDeletedComponentWithDependents`
+in `internal/exec`, panicking on an unexpected error:
+
+```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)